meta-mender-core/classes/mender-install-ubi.bbclass

```diff
@@ -0,0 +1,25 @@
+ARTIFACTIMG_FSTYPE ?= "ubifs"
+
+MENDER_STORAGE_DEVICE ?= "ubi0"
```

Comment:
Actually the `/dev` approach does not work to but in `/etc/fstab`.

```
[   ***] (2 of 2) A start job is running for dev-ubi0_2.device (28s / 1min 30s)[   32.210882] random: nonblocking pool is initialized
[ TIME ] Timed out waiting for device dev-ubi0_2.device.
[DEPEND] Dependency failed for /data.
[DEPEND] Dependency failed for Local File Systems.
[ TIME ] Timed out waiting for device dev-ttymxc0.device.
```

Seems like systemd has an issue with it. `mount -a` works otherwise.